go/weatherapp-gofasthttp/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/rynowak/benchmarks-playground/go/weatherapp-gofasthttp
 go 1.13
 
 require (
+	github.com/fasthttp/router v0.5.2
```

Comment:
should fasthttp-routing be removed from here?

Reply:
Yeah I'm not sure why it wasn't removed when I did go build.. let me check

Reply:
resolved